REPORTING.md

```diff
@@ -38,14 +38,21 @@ A "Missing RECEP Report" can detect these faults.
 
 ## Missing Inbound Transaction
 
-It is possible that an inbound transaction was not processed successfully by the adaptor or lost in transmission. If any
-missing inbound transaction is identified the GP System operator must check the dead-letter queues for any messages that
+It is possible, however unlikely, that an inbound transaction fails to process successfully is lost in transmission. If any
+missing inbound transactions are identified the GP System operator must check the dead-letter queues for any messages that
 could not be processed successfully and correlate with the application logs to determine the cause of the error. If there
-are no matching messages in the dead-letter queues then the GP System operator must ask the NHAIS operator for the reason
-why the message was not received.
+are no matching messages in the dead-letter queues then the GP System operator must ask the NHAIS operator to investigate
+and possibly resend the lost message.
 
 A "Missing Sequence Number Report" on the `inboundState` collection can detect these faults.
 
+## Matching Interchange Sequence Numbers to Operation Ids
+
+It is possible, however unlikely, that the NHAIS operator may call upon the GP System to re-enter transactions due to
```

Comment:
two paragraphs starting 'It is possible, however unlikely, ' will stick out like a sore thumb.  Perhaps we can amend the second instance to 'It is also possible, that...'